backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -186,7 +164,8 @@ private static void displayDockerLog(OutputFrame outputFrame) {
     public void start() {
         if (!cassandraContainer.isRunning()) {
             cassandraContainer.start();
-            administrator().initializeKeyspace(KEYSPACE);
+            administrator().initializeKeyspace(mainKeyspaceConfiguration());
+            administrator().initializeKeyspace(cacheKeyspaceConfiguration());
```

Comment:
we probably don't want to create this keyspace for all tests

Reply:
Modularizing that might be hard. I lack a strategy for it.